tfhe/src/shortint/engine/server_side/comp_op.rs

```diff
@@ -36,24 +36,13 @@ impl ShortintEngine {
         ct_left: &mut Ciphertext,
         ct_right: &mut Ciphertext,
     ) -> EngineResult<Ciphertext> {
-        let mut result = ct_left.clone();
```

Comment:
the above allow does not apply anymore I believe as the message extract assign is performed on ct_left

Reply:
Removed